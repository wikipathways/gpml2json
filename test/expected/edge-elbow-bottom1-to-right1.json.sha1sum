7fa944fc713306880c4728daee17a7bb6c6b6790 ?-
