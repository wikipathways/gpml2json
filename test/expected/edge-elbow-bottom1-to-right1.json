{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["fe51b","a97fd","c3303"],"drawAs":"rect","gpmlElementName":"Pathway","height":177.73163340840773,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":263.5319777888035,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/ff5d99cb6f99008a8cce778966967c8f2bd845dc61c092e295b7926d4a0bed13"},"entityMap":{"a97fd":{"id":"a97fd","textContent":"GeneProduct","wpType":"GeneProduct","x":153.5319777888035,"y":127.73163340840773,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32987,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"c3303":{"id":"c3303","textContent":"0013","wpType":"GeneProduct","x":22.403019470276114,"y":60.37279561407141,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32988,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"fe51b":{"id":"fe51b","drawAs":"ElbowLine","borderWidth":1,"zIndex":32986,"points":[{"isAttachedTo":"c3303","attachmentDisplay":{"position":[0.5,1],"offset":[0,0]},"x":62.403019470276114,"y":80.3727956140714,"orientation":[0,1]},{"y":100.3727956140714,"x":157.9674986295398},{"x":253.5319777888035,"y":119.05221451123957},{"isAttachedTo":"a97fd","attachmentDisplay":{"position":[1,0.5],"offset":[0,0]},"x":233.5319777888035,"y":137.73163340840773,"orientation":[-1,0]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerStart":"TBar","markerEnd":"Arrow","isAttachedTo":["c3303","a97fd"]}}}
